src/controller/java/AndroidDeviceControllerWrapper.h

```diff
@@ -103,6 +108,8 @@ class AndroidDeviceControllerWrapper : public chip::Controller::DevicePairingDel
     void OnScanNetworksSuccess(
         const chip::app::Clusters::NetworkCommissioning::Commands::ScanNetworksResponse::DecodableType & dataResponse) override;
     void OnScanNetworksFailure(CHIP_ERROR error) override;
+    void OnICDRegistrationInfoRequired() override;
```

Comment:
I think we need put detailed doxygen for OnICDRegistrationInfoRequired and OnICDRegistrationComplete here since this is public callback we expect application to override from kotlin and from java.

Reply:
If you check the implementation of this code, it calls a java method and has been added to the CompletionListener interface. Would this code also require the same doxygen?

Reply:
agree, we should put doxygen in the corresponding java methods.